docs/report/report.tex

```diff
@@ -188,6 +200,25 @@ \section{Design Mockups}
 
 \section{Durchführung}
 
+\subsection{Projektstrukur}
+\begin{wrapfigure}{r}{4cm}
+  \includegraphics[width=3.8cm]{PackageStruktur.png}
+  \caption{Orderstruktur}
+  \label{fig:Orderstruktur}
+\end{wrapfigure}
+Die an das Projekt gesetzten Anforderungen machten die Erstellung mehrerer Applikationen notwendig. Neben einer Benutzeroberfläche, auf der die NFC-Tags
+verwaltet werden können, musste die Steuerung des NFC-Readers und ein System zur Speicherung der Daten sowie zum Abspielen der Musik entwickelt werden.
+Die dafür benötigten Anwendungen wurden als Microservices konzipiert und umgesetzt. Zur Verringerung des Aufwands der Konfiguration und Wartung wurden die einzelnen Microservices in einem Mono-Repository auf GitHub zusammengefasst.
+Jeder Service wurde in einem Unterdner in \textit{packages} angelegt (vgl. Abbildung \ref{fig:Orderstruktur}). Der Ordner \textit{app} behinhaltet die Benutzeroberfläche,
+\textit{backed} die Datenbankschnittstelle sowie Logik zur Kommunikation mit dem Musik-Streaming-Anbieter und \textit{nfc-reader} die Applikation zum Steuern und auslesen des am Raspberry Pi angeschlossenen NFC-Readers.
+Der Ordner commons beinhaltet die von allen Services gemeinsam genutzten Ressourcen (wie z. B. Modell-Klassen).
```

Comment:
```suggestion
Jeder Service wurde in einem Unterdner in \textit{packages} angelegt (vgl. Abbildung \ref{fig:Orderstruktur}). Der Ordner \textit{app} enthielt die Benutzeroberfläche,
\textit{backed} die Datenbankschnittstelle sowie Logik zur Kommunikation mit dem Musik-Streaming-Anbieter und \textit{nfc-reader} die Applikation zum Steuern und auslesen des am Raspberry Pi angeschlossenen NFC-Readers.
Im Ordner commons befanden sich die von allen Services gemeinsam genutzten Ressourcen (wie z. B. Modell-Klassen).
```
Sollten die Ordnerinhalte vielleicht auch in Vergangenheitsform? Ist einfach konsistenter zu lesen, auch wenn die Ordner immernoch den Inhalt haben 😅

Reply:
Weiß ich nicht. Also ich bleib da lieber bei Präsenz. Klingt sonst komisch wenn man drüber nachdenkt 😄

Reply:
geändert in  159f14f